miner/coinbase_test.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+from coinbase import serialize_coinbase_transaction
+
+
+class CoinbaseTest(unittest.TestCase):
+    def test_serialize_coinbase_transaction(self):
+        # inspired from: http://bitcoin.stackexchange.com/a/20724
+        # and https://en.bitcoin.it/wiki/Genesis_block
+        reward = 5000000000
```

Comment:
@DrPandemic It's a mix between [the data in this stackexchange question](http://bitcoin.stackexchange.com/questions/20721/what-is-the-format-of-the-coinbase-transaction/20724#20724) (because it was conveniently already serialized) and the [genesis block](https://en.bitcoin.it/wiki/Genesis_block) (because it had simple rewards calculation). Basically, this is hand-crafted to have a simple example to reproduce. Think I should add a comment or something? I thought "inspired by ..." gave a hint that this was hand-crafted.